lib/utils.js

```diff
@@ -1,5 +1,25 @@
 let { list } = require('postcss')
 
+class Cache {
+  constructor () {
+    this.map = new Map()
```

Comment:
`map` should be a "private" field as it shouldn't be exposed, however `private` is only supported since Node.js 12 so I just leave it here.